script/integration-test

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -e
+
+DEST=$1
+
+# subshell so that we can export PATH without breaking other things
+(
+	source "$(dirname "$BASH_SOURCE")/.integration-daemon-start")
```

Comment:
If I'm not mistaken, only one daemon per host can be run, right?

In that case, couldn't just we assume that docker is already running and available at `DOCKER_HOST` and not start it ourselves?